commands/bot.rb

```diff
@@ -58,3 +71,12 @@
   end
 end
 
+hear(/hct/) do
+  clearance(&:sudo?)
+  description 'Shutsdown the bot with an error status'
```

Comment:
s/Shutsdown/Shuts down/. I'd just replace it with a reference to https://en.wikipedia.org/wiki/Halt_and_Catch_Fire